direct/data/sens.py

```diff
@@ -0,0 +1,68 @@
+# coding=utf-8
+# Copyright (c) DIRECT Contributors
+
+from typing import List, Optional, Tuple, Union
+
+import numpy as np
+from scipy.stats import multivariate_normal as normal
+
+
+def simulate_sensitivity_maps(
+    shape: Union[List[int], Tuple[int]], num_coils: int, var: float = 1, seed: Optional[int] = None
+) -> np.ndarray:
+    r"""Simulates coil sensitivities using bi-variate or tri-variate gaussian distribution.
+
```

Comment:
Can you add a reference?

Reply:
There is no reference here It just creates gaussian dristributions with difference centers